pom.xml

```diff
@@ -56,18 +55,27 @@
         <spring.mockmvc.version>1.2.6.RELEASE</spring.mockmvc.version>
     </properties>
 
-    <distributionManagement>
+    <repositories>
         <repository>
-            <id>releases-nexus</id>
-            <name>SDL Releases</name>
-            <url>https://nexus.sdl.com/repository/releases</url>
+            <id>snapshots-repo</id>
+            <url>https://oss.sonatype.org/content/repositories/snapshots</url>
+            <releases><enabled>false</enabled></releases>
+            <snapshots><enabled>true</enabled></snapshots>
         </repository>
-        <snapshotRepository>
-            <id>snapshots-nexus</id>
-            <name>SDL Snapshots</name>
-            <url>https://nexus.sdl.com/repository/snapshots</url>
-        </snapshotRepository>
-    </distributionManagement>
+    </repositories>
+
+<!--    <distributionManagement>-->
```

Comment:
why did you remove this? we might need it for the release process, no?

Reply:
reverted back